src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -335,11 +335,12 @@ namespace smt::noodler {
             expr_ref len(m_util_a.mk_numeral(rational(l), true), m);
 
             literal lit(mk_eq(len_str, len, false));
-            ctx.mark_as_relevant(lit);
+            // ctx.mark_as_relevant(lit);
```

Comment:
Just to check again, is this part of the relevancy problems fix and is this change intended?

Reply:
I think so. The main problem was somewhere else however.

Reply:
That is good enough for me.